### PR TITLE
bug 0011078: 'Default person' instead of 'Home person'

### PR DESCRIPTION
**Pull request by giansalvo**
- modified button hints
- modified filter rules names and descriptions
- modified .pot and .po files

### CODE REVIEW

**Comment by romjerome** [comment]
Note, if you want to push it on gramps51 branch, maybe need to also [modify all translation files](https://gramps-project.org/wiki/index.php/Coding_for_translation#Changing_translated_text_message_in_the_source_code) ...

File: gramps/gen/filters/rules/person/_isdefaultperson.py [hunk 1]
```diff
@@ -41,9 +41,9 @@
 class IsDefaultPerson(Rule):
     """Rule that checks for a default person in the database"""
 
-    name = _('Default person')
+    name = _('Home person')
```
Comment (Nick-Hall) [inline gramps/gen/filters/rules/person/_isdefaultperson.py]: This string has not been updated in the po files.

File: gramps/gen/filters/rules/person/_isdefaultperson.py [hunk 2]
```diff
@@ -41,9 +41,9 @@
...
     category = _('General filters')
-    description = _("Matches the default person")
+    description = _("Matches the home person")
```
Comment (Nick-Hall) [inline gramps/gen/filters/rules/person/_isdefaultperson.py]: Also not updated in the po files.

File: gramps/gen/filters/rules/person/_islessthannthgenerationancestorofdefaultperson.py [hunk 3]
```diff
@@ -44,10 +44,10 @@ class IsLessThanNthGenerationAncestorOfDefaultPerson(Rule):
     not more than N generations away"""
 
     labels = [ _('Number of generations:') ]
-    name = _('Ancestors of the default person '
+    name = _('Ancestors of the home person '
```
Comment (Nick-Hall) [inline gramps/gen/filters/rules/person/_islessthannthgenerationancestorofdefaultperson.py]: Also not updated in the po files.

File: gramps/gen/filters/rules/person/_islessthannthgenerationancestorofdefaultperson.py [hunk 4]
```diff
@@ -44,10 +44,10 @@ class IsLessThanNthGenerationAncestorOfDefaultPerson(Rule):
...
                     'not more than <N> generations away')
     category = _('Ancestral filters')
-    description = _("Matches ancestors of the default person "
+    description = _("Matches ancestors of the home person "
```
Comment (Nick-Hall) [inline gramps/gen/filters/rules/person/_islessthannthgenerationancestorofdefaultperson.py]: Also not updated in the po files.

File: gramps/gui/views/navigationview.py [hunk 5]
```diff
@@ -321,7 +321,7 @@ def home(self, *obj):
         else:
             from ..dialog import WarningDialog
             WarningDialog(_("No Home Person"),
-                _("You need to set a 'default person' to go to. "
+                _("You need to set a 'home person' to go to. "
```
Comment (Nick-Hall) [inline gramps/gui/views/navigationview.py]: Also not updated in the po files.

**Comment by giansalvo** [comment]
> There are still some strings that need updating in the po files.
> 
> Are you using `sed` to do this?

I used the following command
perl -pi -e 's/YOUR MESSAGE BEFORE CORRECTION/your message after correction/g;' *.po *.pot 
as said here
https://gramps-project.org/wiki/index.php/Coding_for_translation#Changing_translated_text_message_in_the_source_code
What's the best command I should use?

**Comment by Nick-Hall** [comment]
You can also use `sed -i 's/find/replace/g' *.po`, but perl is fine.  Scripting the change reduces the chance of errors.

**Comment by prculley** [comment]
Manually rebased, and merged to gramps51.

**Pull request closed without merging**